mods/player_api/api.lua

```diff
@@ -132,7 +132,7 @@ minetest.register_globalstep(function(dtime)
 			elseif controls.LMB then
 				player_set_animation(player, "mine")
 			else
-				player_set_animation(player, "stand", animation_speed_mod)
+				player_set_animation(player, "stand", model.animation_speed / 2)
```

Comment:
We can't do this, as other player models would have their stand animation speed broken.
The standing animation speed would have to be halved in the model itself.
So, i suggest removing this `/ 2` as the rest of the PR is fone.

Reply:
Will do. Other player models? Like in mods?

Reply:
Yes, it's an API so any model can be registered for the player.